README.md

```diff
@@ -11,89 +11,50 @@ https://codecov.io/gh/david26694/cluster-experiments/branch/main/graph/badge.svg
-  - Cluster randomization
-  - Switchback experiments
-- **Dict config**: Easy to configure power analysis with a dictionary
-
-### 2. Experiment Analysis
-- **Analysis Plans**: Define structured analysis plans
-- **Metrics**:
-  - Simple metrics
-  - Ratio metrics
-- **Dimensions**: Slice results by dimensions
-- **Statistical Methods**:
-  - GEE
-  - Mixed Linear Models
-  - Clustered / regular OLS
-  - T-tests
-  - Synthetic Control
-- **Dict config**: Easy to define analysis plans with a dictionary
-
-### 3. Variance Reduction
-- **CUPED** (Controlled-experiment Using Pre-Experiment Data):
-  - Use historical outcome data to reduce variance, choose any granularity
-  - Support for several covariates
-- **CUPAC** (Control Using Predictors as Covariates):
-  - Use any scikit-learn compatible estimator to predict the outcome with pre-experiment data
-
-## Quick Start
-
-### Power Analysis Example
+**`cluster experiments`** is a comprehensive Python library for end-to-end A/B testing workflows, designed for seamless integration with Pandas in production environments.  
 
-```python
-import numpy as np
-import pandas as pd
-from cluster_experiments import PowerAnalysis, NormalPowerAnalysis
+---
 
-# Create sample data
-N = 1_000
-df = pd.DataFrame({
-    "target": np.random.normal(0, 1, size=N),
-    "date": pd.to_datetime(
-        np.random.randint(
-            pd.Timestamp("2024-01-01").value,
-            pd.Timestamp("2024-01-31").value,
-            size=N,
-        )
-    ),
-})
+## 🚀 Key Features  
 
-# Simulation-based power analysis with CUPED
-config = {
-    "analysis": "ols",
-    "perturbator": "constant",
-    "splitter": "non_clustered",
-    "n_simulations": 50,
-}
-pw = PowerAnalysis.from_dict(config)
-power = pw.power_analysis(df, average_effect=0.1)
-
-# Normal approximation (faster)
-npw = NormalPowerAnalysis.from_dict({
-    "analysis": "ols",
-    "splitter": "non_clustered",
-    "n_simulations": 5,
-    "time_col": "date",
-})
-power_normal = npw.power_analysis(df, average_effect=0.1)
-power_line_normal = npw.power_line(df, average_effects=[0.1, 0.2, 0.3])
+### 📌 **Experiment Design & Planning**  
+- **Power analysis** and **Minimal Detectable Effect (MDE)** estimation  
+- Support for **complex experimental designs**, including:  
+  - 🏢 **Cluster randomization**  
+  - 🔄 **Switchback experiments**  
+
+### 🛠 **Data Preprocessing**  
+- Tools for **efficient data preparation**  
+- Seamlessly integrates with **Pandas** for streamlined workflows  
 
+### 📊 **Comprehensive Experiment Analysis**  
+##### **✅ Metrics**  
+- Simple and **ratio-based metrics** for evaluating experiment outcomes  
 
+##### **📈 Statistical Methods**  
+- 📌 **Generalized Estimating Equations (GEE)**  
+- 📌 **Mixed Linear Models** for robust inference  
+- 📌 **Ordinary Least Squares (OLS)** and **Clustered OLS** with covariates  
+- 📌 **T-tests** with variance reduction techniques (**CUPED, CUPAC**)  
```

Comment:
I'd merge this and the one above, and not mention t-tests since mostly its OLS with covariates, cuped, cupac